packages/tokens/src/helpers/getTokensByChain.ts

```diff
@@ -11,3 +11,8 @@ export function getTokensByChain(chainId?: ChainId): Token[] {
   const tokenMap = allTokens[chainId]
   return Object.values(tokenMap)
 }
+
+export function getTokenByAddress(chainId: ChainId, address: SerializedToken['address']) {
+  const tokens = getTokensByChain(chainId)
+  return tokens.find((token) => token.address === address)
```

Comment:
can we just use `isAddressEqual` ?

Reply:
we can. however, the package of @pancakeswap/tokens does not have a dependency on viem, so we can't not use the isAddressEqual. and i think to add a dependency for this case maybe a little heavy. 
I add a toLowercase to safe compare

Reply:
just use `toLowercase` before diff